app/dashboard/migrations/0034_auto_20180129_1344.py

```diff
@@ -0,0 +1,53 @@
+# Generated by Django 2.0 on 2018-01-29 20:44
```

Comment:
This will yield a migration conflict.  Can you delete this file and re-run makemigrations?

Reply:
8200bfe2eeab57ef69fdc108b34fe1fedd47c721